db/migrate/20150812094445_add_email_confirmed_at_to_user.rb

```diff
@@ -0,0 +1,6 @@
+class AddEmailConfirmedAtToUser < ActiveRecord::Migration
+  def change
+    add_column :users, :email_confirmed_at, :date
+    add_column :users, :confirm_token, :string
```

Comment:
Maybe we should do it more like with the password `password_reset_keys` (or use the same table)